apps/visualization/models.py

```diff
@@ -7,7 +10,7 @@ class Narratives(models.Model):
     thematic = models.CharField(max_length=50, default='')
     topic = models.CharField(max_length=50, default='')
     indicator_id = models.CharField(max_length=6, default='')
-    narrative = models.TextField()
+    narrative = HTMLField(verbose_name=_('Narrative'), blank=True, default='')
```

Comment:
Lets not modify model field, add form for narrative and set tinymce in narrative field